frontend/src/scenes/cohorts/Cohort.tsx

```diff
@@ -1,199 +1,300 @@
+                                        <span>
+                                            Actors who match the following criteria will be part of the cohort.
+                                            Continuously updated automatically.
+                                        </span>
+                                    </div>
+                                    {cohort.groups.map((group: CohortGroupType, index: number) => (
+                                        <Group key={index} name={['groups', index]}>
+                                            <div
+                                                style={{
+                                                    display: 'flex',
+                                                    flexDirection: 'column',
+                                                    alignItems: 'center',
+                                                    width: '100%',
+                                                }}
+                                            >
+                                                <KeaField
+                                                    name="id"
+                                                    template={({ error, kids }) => {
+                                                        return (
+                                                            <div
+                                                                style={{
+                                                                    padding: 15,
+                                                                    border: error
+                                                                        ? '1px solid var(--danger)'
+                                                                        : '1px solid rgba(0, 0, 0, 0.1)',
+                                                                    borderRadius: 4,
+                                                                    width: '100%',
+                                                                }}
+                                                            >
+                                                                {kids}
+                                                                <Row>
+                                                                    {error && (
+                                                                        <div
+                                                                            style={{
+                                                                                color: 'var(--danger)',
+                                                                                marginTop: 16,
+                                                                            }}
+                                                                        >
+                                                                            {group.matchType === ENTITY_MATCH_TYPE
```

Comment:
It'd be great if we could push this logic into the form validation, so the message and everything is contained there.